cypress/integration/top_bar_spec.js

```diff
@@ -0,0 +1,39 @@
+import TopBar from '../../js/components/top-bar'
+import simulationStore from '../../js/stores/simulation-store'
+import React from 'react'
+import { Provider } from 'mobx-react'
+import { mount } from '../utils/cypress_react_test'
+
+import css from '../../css-modules/top-bar.less'
+
+describe('Top Bar', function () {
+  beforeEach(() => {
+    mount(<TopBar />)
+  })
+
+  it ('renders correctly', function () {
+    Cypress.component().its('state').should('deep.equal', {
+      shareDialogOpen: false,
+      aboutDialogOpen: false
+    })
+    Cypress.component().then((component) => {
+      const spy = cy.spy(component, 'openAboutDialog')
+      cy.contains('About').click()
+      cy.contains('About').click().then((about) => {
```

Comment:
Again, I expected to be able to perform a single click and then check the spy in the next line. For some reason, I needed to perform two clicks, *and* wrap the spy check in a `then`.